python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -196,8 +196,11 @@ def last(self, split_every=None, split_out=1):
     @_deprecate_shuffle_kwarg
     @_dask_cudf_nvtx_annotate
     def aggregate(
-        self, arg, split_every=None, split_out=1, shuffle_method=None
+        self, arg, split_every=None, split_out=1, shuffle_method=None, **kwargs
     ):
+        if kwargs:
+            arg = {col_name: agg_func for col_name, (col, agg_func) in kwargs.items()}
+
```

Comment:
I know you are already revising this PR, but just a quick note: These changes are being made to the legacy dask-cudf API (which will soon be deprecated and then completely removed). The new (default) expression-based API is defined [in expr/_groupby.py](https://github.com/rapidsai/cudf/blob/fa8284ddb2de808573d5b21cc9e650578ddf6acc/python/dask_cudf/dask_cudf/expr/_groupby.py#L92).

Reply:
Ok Sure working on cudf now